$NetBSD$

Disable the arch check, which never works (at least on SunOS).
--- storage/tokudb/CMakeLists.txt.orig	2015-02-25 15:40:56.000000000 +0000
+++ storage/tokudb/CMakeLists.txt
@@ -1,5 +1,5 @@
 # ft-index only supports x86-64 and cmake-2.8.9+
-IF(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64" AND
+IF(
     NOT CMAKE_VERSION VERSION_LESS "2.8.9")
 CHECK_CXX_SOURCE_COMPILES(
 "
